cloud/deployment/fromfile/fromfile.go

```diff
@@ -327,12 +327,14 @@ func createOrUpdateDeployment(deploymentFromFile *inspect.FormattedDeployment, c
 
 			var schedulerSize astroplatformcore.CreateStandardDeploymentRequestSchedulerSize
 			switch strings.ToUpper(deploymentFromFile.Deployment.Configuration.SchedulerSize) {
-			case deployment.SMALL:
+			case string(astrocore.CreateStandardDeploymentRequestSchedulerSizeSMALL):
```

Comment:
it's a case of "small" vs "SMALL" 🙈, we are taking "small" as user input in CLI and API expects "SMALL"

Reply:
So:
```
switch strings.ToLower(deploymentFromFile.Deployment.Configuration.SchedulerSize):
case deployment.SmallScheduler:
```
?

Reply:
I might prefer relying on the auto-generated enums from the Core API client package, for comparing against the Core API client response, just to avoid any unwanted harm down the line. But I agree the wording is way too verbose.

Reply:
I'd expect that since we already removed `deployment.SMALL` etc we'd also remove `smallScheduler` etc. Yeah it's verbose with the `strings.ToLower` added on but it's closer to the truth of what is a valid input than redefining it as a constant.

Reply:
updated the logic